docs/source/developers/contributing/prerequisites.rst

```diff
@@ -73,9 +69,11 @@ Our integration/multicloud tests perform real requests to cloud providers. You c
 
 Note that the ``make test`` command only runs unit tests.
 
-If you would like to run these locally, you will need to be authenticated. The following are the steps for each respective cloud provider.
+If you would like to run these locally, you will need to be authenticated.
+The following are the steps for each respective cloud provider.
 
-**Google Cloud**
+Google Cloud
+^^^^^^^^^^^^
```

Comment:
In the below, I am confused by the second bullet under the last numbered point. Isn't it that you choose this if you want to not delete the project? I think the beginning of the point doesn't really capture that. I would then also specifically say that you must then delete everything on your own, and here is the order to do that.

Reply:
Not exactly. For not deleting the project, we have `PARAGLIDER_TEST_PERSIST` which I have in the section below. If you use `PARAGLIDER_GCP_PROJECT`, then the project by default isn't deleted (regardless of the value of `PARAGLIDER_TEST_PERSIST`. I added some more wording. Let me know if that's still confusing.

<img width="2032" alt="Screenshot 2024-07-11 at 23 37 02" src="https://github.com/user-attachments/assets/8150e217-7c3a-4b59-b29b-09e4f8c70b20">